src/main/java/racingcar/car/CarType.java

```diff
@@ -0,0 +1,5 @@
+package racingcar.car;
+
+public enum CarType {
+    SMALL, Medium, Large
```

Comment:
ENUM 선언을 할 때에는 모두 대문자로 선언해주세요~
```suggestion
    SMALL, MEDIUM, LARGE
```

Reply:
아차차.. 반영하겠습니다 !